sample-code/src/ai-api/deployment-api.ts

```diff
@@ -125,3 +126,23 @@ export async function deleteDeployments(
     )
   );
 }
+
+/**
+ * Get the deployment URL for a given model and scenario.
+ * @param model - Model name.
+ * @param scenarioId - Scenario ID.
+ * @param resourceGroup - AI-Resource-Group where the resources are available.
+ * @returns The deployment URL.
+ */
+export async function getDeploymentUrl(
+  model: string,
+  scenarioId: string,
+  resourceGroup: string
```

Comment:
should be optional and default to `default`. Compare with what is optional in `DeploymentResolutionOptions`